features/counters.yml

```diff
@@ -0,0 +1,17 @@
+name: Counters (CSS)
+description: The `counter()` and `counters()` CSS functions, used with `counter-reset` and `counter-increment`, automatically number headings or ordered list items.
+spec: https://drafts.csswg.org/css-lists-3/#auto-numbering
+caniuse: css-counters
+group: counters
+status:
+  compute_from: css.types.counter
```

Comment:
We ought to avoid pinning the status to something that predates an explicit item in the description. Can you pin to the properties instead?